src/App/Mail/SendTodoReminderMailSubscriber.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Prooph\ProophessorDo\App\Mail;
+
+use Prooph\ProophessorDo\Model\Todo\Event\TodoAssigneeWasReminded;
+use Prooph\ProophessorDo\Projection\Todo\TodoFinder;
+use Prooph\ProophessorDo\Projection\User\UserFinder;
+
+/**
+ * Class SendTodoReminderMailSubscriber
+ *
+ * @package Prooph\ProophessorDo\App\Mail
+ * @author Roman Sachse <r.sachse@ipark-media.de>
+ */
+class SendTodoReminderMailSubscriber
+{
+    /**
+     * @var UserFinder
+     */
+    private $userFinder;
+    /**
+     * @var TodoFinder
+     */
+    private $todoFinder;
+
+    /**
+     * @param UserFinder $userFinder
+     * @param TodoFinder $todoFinder
+     */
+    public function __construct(UserFinder $userFinder, TodoFinder $todoFinder)
+    {
+        $this->userFinder = $userFinder;
+        $this->todoFinder = $todoFinder;
+    }
+
+    public function __invoke(TodoAssigneeWasReminded $event)
+    {
+        $user = $this->userFinder->findById($event->userId()->toString());
+        $todo = $this->todoFinder->findById($event->todoId()->toString());
+
+        $mail = "Hello {$user->name}. This a reminder for {$todo->text}";
```

Comment:
yes why not

Sascha-Oliver Prolic

2016-03-15 14:43 GMT+00:00 Roman Sachse <notifications@github.com>:

> In src/App/Mail/SendTodoReminderMailSubscriber.php
> <https://github.com/prooph/proophessor-do/pull/73#discussion_r56174121>:
>
> > +    /**
> > +     * @param UserFinder $userFinder
> > +     * @param TodoFinder $todoFinder
> > +     */
> > +    public function __construct(UserFinder $userFinder, TodoFinder $todoFinder)
> > +    {
> > +        $this->userFinder = $userFinder;
> > +        $this->todoFinder = $todoFinder;
> > +    }
> > +
> > +    public function __invoke(TodoAssigneeWasReminded $event)
> > +    {
> > +        $user = $this->userFinder->findById($event->userId()->toString());
> > +        $todo = $this->todoFinder->findById($event->todoId()->toString());
> > +
> > +        $mail = "Hello {$user->name}. This a reminder for {$todo->text}";
>
> Currently yes. How should I approach this? Usinghttp://
> framework.zend.com/manual/current/en/modules/zend.mail.introduction.html?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/prooph/proophessor-do/pull/73/files#r56174121
>